packages/react/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -74,7 +74,7 @@ export const ThirdwebProvider = <
           supportedWallets={
             supportedWallets || [
               metamaskWallet(),
-              coinbaseWallet(),
+              coinbaseWallet({headlessMode: true}),
```

Comment:
`headlessMode` must always be `true` when using ConnectWallet, so need to move it to the definition of `coinbaseWallet` function and omit the `headlessMode` from config type